web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -459,15 +457,15 @@
   };
 
   const onStackAssets = async () => {
-    const ids = await stackAssets(Array.from($selectedAssets));
+    const ids = await stackAssets(assetInteraction.selectedAssetsArray);
     if (ids) {
       $assetStore.removeAssets(ids);
       onEscape();
     }
   };
 
   const toggleArchive = async () => {
-    const ids = await archiveAssets(Array.from($selectedAssets), !isAllArchived);
+    const ids = await archiveAssets(assetInteraction.selectedAssetsArray, assetInteraction.isAllArchived);
```

Comment:
You missed the *not* here, right? `!assetInteraction.isAllArchived`

Reply:
Yup, well spotted